src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,7 +5,14 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private String model;
+
+    public Truck(String model) {
+        this.model = model;
+    }
+
     public Truck() {
```

Comment:
The no-args constructor is present, which is good as it is needed to run the test. However, consider removing the unnecessary empty line within the constructor body.